### PR TITLE
net/rs-aggregate: submission (new Portfile)

### PR DESCRIPTION
**Pull request by herrbischoff**
###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on
macOS 14.1.2 23B92 x86_64
Command Line Tools 15.0.0.0.1.1694021235

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint --nitpick`?
- [x] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: net/rs-aggregate/Portfile [hunk 1]
```diff
@@ -0,0 +1,219 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cargo 1.0
+
+github.setup        ktims rs-aggregate v0.3.2
```
Comment (herbygillot) [inline net/rs-aggregate/Portfile:7]: The format of `github.setup` should be `<author> <project> <version> <prefix> <postfix>`

So this should be:

```suggestion
github.setup        ktims rs-aggregate 0.3.2 v
```

File: net/rs-aggregate/Portfile [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+github.tarball_from archive
+categories          net
+platforms           darwin
```
Comment (herbygillot) [inline net/rs-aggregate/Portfile:10]: `platforms darwin` is implicit in MacPorts now.  You can remove this line.

File: net/rs-aggregate/Portfile (hunk 1 above)
Comment (herrbischoff) [reply net/rs-aggregate/Portfile]: Thanks for the heads-up!

File: net/rs-aggregate/Portfile (hunk 2 above)
Comment (herrbischoff) [reply net/rs-aggregate/Portfile:10]: Will do, thanks!

**Comment by pmetzger** [comment]
@herbygillot Are you happy with this now?

**Comment by herbygillot** [comment]
Merged, thanks for your submission, @herrbischoff.

**Comment by herrbischoff** [comment]
Thanks for merging – and on a Sunday no less.

**Pull request merged**